lib/msal-browser/docs/v4-migration.md

```diff
@@ -87,6 +98,9 @@ pca.handleRedirectPromise({
     ```typescript
       pca.handleRedirectPromise({ navigateToLoginRequestUrl: false })
     ```
+1. The `encodeExtraQueryParams` parameter has been removed. All extra query params will be encoded.
+1. The `supportsNestedAppAuth` parameter has been removed. Use `createNestablePublicClientApplication()` instead.
```

Comment:
Same comment here r.e. showing Before and After usage

Reply:
Added